docs/source/markdown/podman-kube-play.1.md.in

```diff
@@ -233,7 +233,7 @@ Pods removed:
 `podman kube play --down` will not work with a URL if the YAML file the URL points to
 has been changed or altered.
 
-
+@@option tls-verify
```

Comment:
@rhatdan do you have any idea why you added this here? It's weird and out of place, I'm 99.99% sure it's an oops. I am going to remove it in my upcoming include-mechanism PR.

Reply:
SGTM